### ISSUE TITLE
Not valid cards

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#83]
I've compared the output of `card list ATTokenNewSpec2 -v` with `card list ATTokenNewSpec2` and it seems like we have one not valid transaction:
```
| f4e26f3a14a7fbe7ea45bd60ad9005ebbaa36a3cecd864c7f3ef517c5fe27c86 | 239047   | 1   | mkfuTykT6anzsRjYH7Lktc2hFuTqcGpha7 | mixcqdAy4aFtuBV7uRebqBTjiFWCcS4UEp | 50000.0  | CardIssue    |
```
Does it sound right to you?

**Comment by d5000** [comment]
This can happen, I also did tests with purposefully invalid CardIssues and CardTransfers, but in this case it seems it is a bug. 

f4e26f3a14a7fbe7ea45bd60ad9005ebbaa36a3cecd864c7f3ef517c5fe27c86 is a PoB token claim transaction with 3 claims (100000 + 50000 + 50000) to different addresses. Each of these claims (CardIssues) is validated separately, in the order it was entered. These transfers are also shown separately in `card list`.

In theory the transaction should be valid completely (all CardIssues valid) because it references a burn transaction with 2000 coins, and the multiplier of this token is 100 (so the user is entitled to claim 200000). It seems however that for some reason the algorithm counted one output of this transaction twice, and this led the last claim (which should be valid) to be invalidated because this means that in total 250000 coins were claimed.

Added to my ToDo list, will continue the thread when fixed (or found an explanation for the behaviour). Thanks for the report!
